build_tools/third_party/tint/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ include(FetchContent)
 FetchContent_Declare(
   tint
   GIT_REPOSITORY https://dawn.googlesource.com/tint
-  GIT_TAG 874d1b66186ff8d0cb249bc73ccaea08144c36b0 # 2022-10-21
+  GIT_TAG 30d45f72d562efda45f3b63b29cc05918138d439 # 2022-11-28
```

Comment:
> There are new errors with `exp`, `log`, `sigmoid`: `error: value cannot be represented as 'f32': -inf`

Eek, that error broke resnet, and posenet is also still broken (https://github.com/iree-org/iree/issues/11229 ?):

```
Tint reported 1 error(s) for a SPIR-V program, see diagnostics:
error: line:36: Structure id 8 decorated as Block for variable in Uniform storage class must follow relaxed uniform buffer layout rules: member 0 contains an array with stride 4 not satisfying alignment to 16
  %_struct_8 = OpTypeStruct %_arr_uint_uint_2
```

I'll investigate a bit and maybe file some new issues...

Reply:
Okay, the posenet error is unrelated to Tint updates, but the `inf` error is concerning if real programs (and not just some op unit tests) are affected

Reply:
Filed https://github.com/iree-org/iree/issues/11321 for the `inf` related errors